values-minio.yaml

```diff
@@ -2,4 +2,9 @@ accessKey: admin
 secretKey: password
 defaultBucket:
   enabled: true
-  name: vvp
\ No newline at end of file
+  name: vvp
+
+resources:
+  requests:
+    memory: 256Mi
+    cpu: 250m
```

Comment:
Should we rather use the previous defaults of 100m here because we are already a bit tight on CPU?

Reply:
Scratch that, I only looked at the doc's diff and didn't see it was out of sync.